src/plugins/slack/tests/fetch.test.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Is the flow annotation required to be at the top of the file? without just the tests default to "js-dom" env, since that is what is set in the package.json, without explicitly telling about the node env, the headers are incorrectly transferred to Slack's web API

Reply:
@nglglhtr  The flow pragma is indeed required. I'm confused about your question, though. How does this pragma cause issues with Slack? If you're getting flow errors, we'll have to find a way to resolve them.